src/openms/include/OpenMS/FORMAT/HANDLERS/MzMLHandler.h

```diff
@@ -438,6 +438,8 @@ namespace OpenMS
       Map<String, Software> software_;
       /// The data processing list: id => Instrument
       Map<String, Instrument> instruments_;
+      /// CV terms that have been checked in validateCV_()
+      Map<std::pair<String, String>, bool> cached_terms_;
```

Comment:
if you make this `mutable` you can add back the `const` to the member functions...